swaps/rebalance.js

```diff
@@ -596,6 +598,8 @@ module.exports = (args, cbk) => {
           return cbk([400, 'ExpectedDifferentPeersForInboundAndOutbound']);
         }
 
+        const maxFee = !!args.max_fee_rate ? round((args.max_fee_rate * tokens) / 1e6) : round((defaultMaxFeeRate * tokens) / 1e6);
```

Comment:
ah this wasn't changed to mtokens? I think max fee has to be altered to max_fee_mtokens to gain more precision

here is an example of how to calculate the fee mtokens: https://github.com/alexbosworth/bolt07/blob/master/routing/policy_fee.js

Reply:
i think it has to happen here because otherwise the round will lose precision

Reply:
Can you check now? I added another arg called max_fee_mtokens because if anything else uses max_fee for probing using same file won't break because we are now passing mtokens.
And added a check to pass either max_fee or max_fee_mtokens and can't pass both.

Reply:
wouldn't it be a problem if the max fee was set but was using 0?

I don't know if the check is super necessary, getRouteToDestination is the method that is ultimately called and it already does have a check to make sure you didn't put both max fee and max fee mtokens

Reply:
https://github.com/alexbosworth/balanceofsatoshis/blob/master/swaps/rebalance.js#L110

https://github.com/alexbosworth/balanceofsatoshis/blob/master/swaps/rebalance.js#L114


0 is not supported anyway from before, so it won't matter I guess unless you want to change it now?

Reply:
execute probe is called by other things though

Reply:
so just changing that condition to `args.max_fee_rate !== undefined` it should work I guess? Rest should just work as `round(0)` is just 0?

Reply:
what makes it required exactly?